src/libraries/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -234,6 +232,12 @@ public void GetFullPath_InvalidArgs()
                     { root + new string(Path.DirectorySeparatorChar, 3), root },
                 };
 
+                if (!PlatformDetection.IsBrowser)
```

Comment:
nit: we could make this more generic by checking whether we're at the root:

```suggestion
                if (currentDirectory != Path.GetPathRoot(currentDirectory))
```

Reply:
Done